src/main/java/de/zalando/aruha/nakadi/controller/EventPublishingController.java

```diff
@@ -61,14 +68,23 @@ public ResponseEntity postEvent(@PathVariable final String eventTypeName, @Reque
         try {
             final EventType eventType = eventTypeRepository.findByName(eventTypeName);
 
-            final JSONObject eventAsJson = parseJson(event);
+            try {
+                final Timer successfullyPublishedTimer = metricRegistry.timer(metricNameFor(eventTypeName, SUCCESS_METRIC_NAME));
```

Comment:
I'm not sure about this place. Will it create object with every call? Can we create it only one time?